lib/event_summarizer/vendor_result_evaluators/instant_verify.rb

```diff
@@ -0,0 +1,83 @@
+# frozen_string_literal: true
+
+module EventSummarizer
+  module VendorResultEvaluators
+    module InstantVerify
+      # @param result {Hash} The result structure logged to Cloudwatch
+      # @return [Hash] A Hash with a type and description keys.
+      def self.evaluate_result(result)
+        if result['success']
+          return {
```

Comment:
as a personal style thing, I try to avoid the `return` keyword in Ruby in methods like these where the control flow is fairly straightforward? I think we could accomplish the same result in this method with `if/elsif/elsif/else` kinda thing